src/main/java/xolpoc/config/DeployerConfiguration.java

```diff
@@ -50,13 +67,30 @@ public ModuleDeployer moduleDeployer() {
 
 	@Bean
 	public ResourceModuleRegistry moduleRegistry() {
-		return new ResourceModuleRegistry(MODULE_HOME);
+		return new ResourceModuleRegistry(deployer.getModule());
 	}
 
 	@Bean
 	public ModuleFactory moduleFactory() {
 		return new ModuleFactory(moduleOptionsMetadataResolver());
 	}
+	
+	@Bean
+	@ConfigurationProperties("property")
+	public ModuleDeploymentProperties moduleDeploymentProperties() {
+		return new ModuleDeploymentProperties();
```

Comment:
That implies the `IntegrationEvaluationContextAwareBeanPostProcessor` is missing, which implies no `@EnableIntegration`.